archive/progress_http/src/main/scala/uk/ac/wellcome/platform/archive/progress_http/model/ErrorResponse.scala

```diff
@@ -0,0 +1,6 @@
+package uk.ac.wellcome.platform.archive.progress_http.model
+
+case class ErrorResponse(httpStatus: Int,
```

Comment:
This doesn't quite match the errors we return from the catalogue API, which have an `errorType` of `http`.

We should probably put those models in a common lib.

Reply:
yeah, I'm not sure what's the point of the errorType field if I'm honest. Rather than putting it in hardcoded I went for leaving it out. We can always add it later. I'm not sure if they should be in the same lib though. I can see those models eventually diverging